docs/sources/http_api/datasource_permissions.md

```diff
@@ -23,6 +26,16 @@ The permission levels for the permission field:
 
 Enables permissions for the data source with the given `id`. No one except Org Admins will be able to query the data source until permissions have been added which permit certain users or teams to query the data source.
 
+### Required permissions
+
+See note in the [introduction]({{< ref "#data-source-permissions-api" >}}) for an explanation.
+
+| Action                         | Scope                                                                        |
+| ------------------------------ | ---------------------------------------------------------------------------- |
+| datasources.permissions:toggle | datasources:\*<br>datasources:id:\*<br>datasources:id:1 (single data source) |
```

Comment:
Is it `datasources.permissions` or `datasources:permissions`? I see the second one used in `permissions.md`.

Reply:
Good catch thanks!